src/install.rs

```diff
@@ -502,7 +502,18 @@ fn stash_saved_partitions(disk: &mut File, saved: &SavedPartitions) -> Result<()
 fn is_dasd(config: &InstallConfig) -> Result<bool> {
     let target = canonicalize(&config.device)
         .chain_err(|| format!("getting absolute path to {}", config.device))?;
-    Ok(target.to_string_lossy().starts_with("/dev/dasd"))
+    if !target.to_string_lossy().starts_with("/dev/dasd") {
```

Comment:
Minor: you can move this `.to_string_lossy()` directly into `target` definition, that should spare you the `.display()` a few line below (same applies to `name` too).

Reply:
Did it for `name`. From `target` have to extract `file_name`, so couldn't convert it to string

Reply:
Oh indeed, sorry I missed that aspect.